### PR TITLE
Design a survey

### PR DESCRIPTION
**Pull request by retabak**

### CODE REVIEW

**Comment by retabak** [comment]
@schanzer - do you want to take a look at this before I merge? It's basically identical to the project you wrote. No real changes needed except for formatting it to be a lesson.

File: lessons/Projects/project-design-a-survey/langs/en-us/index.adoc [hunk 1]
```diff
@@ -61,18 +56,26 @@ Now that you know what questions your team wants to ask, it's time to actually c
...
-* At least 8 good questions, copied from your @opt-printable-exercise{survey-brainstorm.adoc}.}
 
-Once a rough draft has been completed, students are ready to make improvements to the survey so that it yeilds the cleanest results possible!
+* At least 8 good questions, copied from your @handout{survey-brainstorm.adoc}.}
+
+Once a rough draft has been completed, students are ready to make improvements to the survey so that it yeilds the cleanest results possible.
 
 @lesson-instruction{
 Read through the survey with your team, considering each of the questions one by one, and making improvements along the way.
+
```
Comment (schanzer) [inline lessons/Projects/project-design-a-survey/langs/en-us/index.adoc:70]: @retabak Why do these need to have extra blank spaces?

File: lessons/Projects/project-design-a-survey/langs/en-us/pages/survey-hacking.adoc [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+@fitb{}{}
+
+@n For each dirty data example, write down what you submitted and what suggestion you would have to help the other group guard against it.
+
+[cols="1a,^4a,^6a", options="header"]
+|===
+| Question Number
+| Dirty Data Entered
+| Suggestion for Improving the Survey Question
```
Comment (schanzer) [inline lessons/Projects/project-design-a-survey/langs/en-us/pages/survey-hacking.adoc:15]: Why not use `.FillVerticalSpace` here?

**Review by schanzer — CHANGES_REQUESTED** [review]
Looks great! Some small formatting questions, but otherwise this is ready to go

File: lessons/Projects/project-design-a-survey/langs/en-us/pages/survey-hacking.adoc (hunk 2 above)
Comment (retabak) [reply lessons/Projects/project-design-a-survey/langs/en-us/pages/survey-hacking.adoc:15]: I think you must've made this page before `.FillVerticalSpace` was around. I'll fix.

File: lessons/Projects/project-design-a-survey/langs/en-us/index.adoc (hunk 1 above)
Comment (retabak) [reply lessons/Projects/project-design-a-survey/langs/en-us/index.adoc:70]: @schanzer - not sure what you mean? 

if I remove the blank lines (is that what you're talking about?), this is what happens:

image

Comment (schanzer) [reply lessons/Projects/project-design-a-survey/langs/en-us/index.adoc:70]: No, that's what happens if you get rid of the _FIRST_ blank line. In asciidoc, lists always need a blank line before they start.

The rest of the blanks are unecessary

**Comment by retabak** [comment]
Whoa. I did NOT know that. How did I not know that?! Thanks.

**Pull request merged**